apps/worker/src/app/workflow/usecases/send-message/send-message.base.ts

```diff
@@ -48,16 +49,16 @@ export abstract class SendMessageBase extends SendMessageType {
   }
 
   @CachedQuery({
-    builder: ({ environmentId, ...command }: GetDecryptedIntegrationsCommand) =>
+    builder: ({ environmentId, ...command }: GetEnvironmentDecryptedIntegrationsCommand) =>
```

Comment:
In the worker, the functionalities regarding the providers should still be tackled by environment, while the functionalities in the API for the general endpoints (get integrations and get active integrations) need to be done at organization level. This caused to have to duplicate the commands used for getting the decrypted integrations to satisfy the conditions without having to create an extra use case like `GetEnvironmentDecryptedIntegrationsUseCase`. That could be an option we can follow if preferred.

Reply:
I feel like we don't need to have two commands, just make the env property optional... we don't use anywhere that approach with two commands that might be provided to the use-case, this is confusing...

Reply:
TypeScript complains if you provide `environmentId` as optional because it is needed for the `buildIntegrationKey` functionality. 😕 
Therefore the reason for duplication. I am just proposing 2 different ways. At first went for the lazier one.

Reply:
this decorator will be removed after Pawels & Biswa PRs merged, and we will cache the `select integration usecase`.
will it solve the command duplication here?

Reply:
this decorator will be removed after Pawels & Biswa PRs merged, and we will cache the select integration usecase.
will it solve the duplication here?

Reply:
Yeah, it could be. But this PR might need to go before so then this is temporary and then deleted on their work.